style.css

```diff
@@ -294,6 +279,97 @@ main {
     text-decoration: none;
   }
 
+  .mobile-menu {
+    position: fixed;
+    width: 375px;
+    height: 745px;
+    background-color: #6070ff;
+    text-align: left;
+    transition: 0.3s;
+    mix-blend-mode: multiply;
+    opacity: 1;
+    backdrop-filter: blur(7px);
+    left: -100%;
+  }
+
+  .mobile-menu.active {
+    left: 202px;
+    position: fixed;
+  }
+
+  .menu-bar {
+    position: absolute;
+    width: 27px;
+    height: 718px;
+    left: 340px;
+    top: -31px;
+    align-self: flex-end;
+    border: 0;
+    outline: 0;
+  }
+
+  .menu-bar.active {
+    opacity: 0;
+  }
```

Comment:
- [ ] **Good job so far 👏** However, I think it would be better to your mobile version to make the body of the page to fit the width of the device for the best user experience and also to replica the Figma design 👍 👍 

You may use the ```width: 100%``` or ```width: 100vw;``` to avoid empty spaces around your content 🙏 

![image](https://user-images.githubusercontent.com/102757126/223061144-23fa3c36-99fe-48c3-9b6a-f0cc73777414.png)

Reply:
- [ ] Kindly implement the suggestions of my colleague on the appearance of your mobile view in order to provide a satisfying experience to users or visitors. The page seems to not be correctly positioned due to some absolute declarations of `width: 375px` and some other extraneous use of `margins`. Kindly watch this [video](https://www.awesomescreenshot.com/video/15356764?key=f04e45a87ab54b58da3677c098f868dd) to have a catch of what I am referring to, just like he mentioned before: